scripts/fortran_tools/offline_check_fortran_vs_metadata.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+"""
+Recursively compare all fortran and metadata files in user-supplied directory, and report any problems
+USAGE: set PYTHONPATH environment variable to the the scripts directory, then:
+    python offline_check_fortran_vs_metadata.py --directory <full path to directory with scheme files> (--debug)
+"""
+
+
+import sys
+import os
+import logging
+import argparse
+
+# CCPP framework imports
+from framework_env import CCPPFrameworkEnv
+from fortran_tools import parse_fortran_file
+from metadata_table import parse_metadata_file
+from ccpp_capgen import find_associated_fortran_file
+from ccpp_capgen import check_fortran_against_metadata, parse_scheme_files
+from parse_tools import init_log, set_log_level
+from parse_tools import register_fortran_ddt_name
+from parse_tools import CCPPError, ParseInternalError
+
+_LOGGER = init_log(os.path.basename(__file__))
+_DUMMY_RUN_ENV = CCPPFrameworkEnv(_LOGGER, ndict={'host_files':'',
+                                                    'scheme_files':'',
+                                                    'suites':''})
+_CCPP_FRAMEWORK_DDT_TYPES = ["ccpp_hash_table_t",
+                             "ccpp_hashable_t",
+                             "ccpp_hashable_char_t",
+                             "ccpp_constituent_prop_ptr_t"]
+
+
+
+def find_files_to_compare(directory):
+    metadata_files = []
+    for root, _, files in os.walk(directory, topdown=True):
```

Comment:
Can't you use the `glob.glob` function with `recursive=True` here?
```
glob.glob(os.path.join(directory,'**','*.meta'), recursive=True)
```

Reply:
ooh yes. changed.